### DISCUSSION TITLE
login background image 403 for private wiki

### DISCUSSION THREAD

**Discussion by cinatic** [requarks/wiki#4368] [Error / Bug Report] [answered]
i uploaded  a custom login background image in the settings and selected it from the media browser. But it is not shown, the request for this images returns a 403, is that an issue when wiki is set to private maybe? Changing it directly with a public available url is working

**Comment by NGPixel** [comment, marked as answer]
You need to allow the background image path to be viewed by guests. Make sure the Guests group has read access to the path where you uploaded the image.
